lib/ansible/modules/source_control/gitlab_deploy_key.py

```diff
@@ -1,43 +1,57 @@
 #!/usr/bin/python
-# -*- coding: utf-8 -*-
-
-# (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
+# Copyright: (c) 2019, Guillaume Martinez (guillaume.lunik@gmail.com)
+# Copyright: (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
 # Based on code:
-# (c) 2013, Phillip Gentry <phillip@cx.com>
+# Copyright: (c) 2013, Phillip Gentry <phillip@cx.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
----
 module: gitlab_deploy_key
 short_description: Manages GitLab project deploy keys.
 description:
      - Adds, updates and removes project deploy keys
 version_added: "2.6"
 options:
-  api_url:
+  server_url:
     description:
-      - GitLab API url, e.g. https://gitlab.example.com/api
+      - Url of Gitlab server, with protocol (http or https).
     required: true
-  access_token:
+    version_added: "2.8"
+    aliases:
+      - api_url
+  verify_ssl:
+    description:
+      - When using https if SSL certificate needs to be verified.
+    type: bool
+    default: 'yes'
```

Comment:
These days we don't expect this to be strings, we now generate documentation correctly. So this can be real booleans.
```suggestion
    default: yes
```

Reply:
Will be fix in the next commit